### PR TITLE
Added new badge and message for AI kit

### PR DESCRIPTION
**Pull request by ErikCH**
#### Description of changes:

This creates a new badge and message about AI kit. This will only appear on the homepage.

image

And a new New badge:

image

Dark mode

image

image
 

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which platform(s) are affected by this PR (if applicable)?
- [x] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [x] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [x] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [x] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by dbanksdesign — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -350,24 +350,44 @@
     fast-xml-parser "^4.4.1"
     tslib "^2.5.0"
 
-"@aws-amplify/ui-react-core@3.0.22":
-  version "3.0.22"
-  resolved "https://registry.npmjs.org/@aws-amplify/ui-react-core/-/ui-react-core-3.0.22.tgz"
-  integrity sha512-uL5jspqvTZhpqH1inPV3ifvrzIVgIIriXPgjz4BaceDm+1X03Hc3tfq5TiUKW8PdpuWF6riXXBP3MEFsk29OGw==
+"@aws-amplify/ui-react-ai@^1.0.0":
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply yarn.lock]: I updated the yarn.lock to match main

File: src/directory/directory.d.ts [hunk 2]
```diff
@@ -57,4 +57,9 @@ export type PageNode = {
    * This is being used for categories like Cli - Legacy and SDK
    */
   hideChildrenOnBase?: boolean;
+
+  /**
+   * This flag indicates that the item is new and will display a pill
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply src/directory/directory.d.ts]: 👍

File: yarn.lock [hunk 3]
```diff
@@ -376,10 +396,34 @@
...
+    "@aws-amplify/ui-react-core" "3.1.0"
+    "@radix-ui/react-direction" "1.0.0"
+    "@radix-ui/react-dropdown-menu" "1.0.0"
+    "@radix-ui/react-slider" "1.0.0"
+    "@xstate/react" "^3.2.2"
+    lodash "4.17.21"
+    qrcode "1.5.0"
+    tslib "^2.5.2"
+
+"@aws-amplify/ui@6.6.6", "@aws-amplify/ui@^6.6.6":
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply yarn.lock]: I updated the yarn.lock to match main

File: src/data/gen2Banner-urls.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+/**
+ * List of paths that the gen2 <Banner /> should display on
+ */
+export const GEN2BANNER_URLS = ['/'];
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply src/data/gen2Banner-urls.ts]: I removed this file, and refactored

File: src/components/Banner/Banner.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import { Flex, Message, IconsProvider, Text } from '@aws-amplify/ui-react';
+import { IconStar, IconChevron } from '../Icons';
+import { Button } from '@aws-amplify/ui-react';
+
+interface BannerProps {
+  url?: string;
+}
+
+export const Banner: React.FC<BannerProps> = ({
+  url = '/react/ai/set-up-ai/'
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply src/components/Banner/Banner.tsx:10]: OK I'll make it a constant variable URL

File: src/components/Banner/Banner.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+              experience in cloud architecture or AI needed.
+            </Text>
+          </Flex>
+
+          <Button
+            as="a"
+            href={url}
+            size="small"
+            gap="small"
+            target="_blank"
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply src/components/Banner/Banner.tsx:37]: Fixed

File: src/components/Layout/Layout.tsx [hunk 7]
```diff
@@ -71,6 +72,7 @@ export const Layout = ({
   const basePath = 'docs.amplify.aws';
   const metaUrl = url ? url : basePath + asPathWithNoHash;
   const pathname = router.pathname;
+  const shouldShowGen2Banner = asPathWithNoHash === '/';
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply src/components/Layout/Layout.tsx:75]: Updated

**Review by dbanksdesign — APPROVED** [review]

**Pull request merged**